pipeline/trusted-resources/pkg/trustedtask/trustedtaskrun.go

```diff
@@ -95,16 +102,19 @@ func (tr *TrustedTaskRun) verifyTaskRun(
 	}
 
 	// TODO: figure out how to config this.
+
 	cfg := config.FromContextOrDefaults(ctx)
 	cfg.FeatureFlags.EnableTektonOCIBundles = true
 	ctx = config.ToContext(ctx, cfg)
 
 	fn, err := resources.GetTaskFunc(ctx, k8sclient, tektonClient, tr.Spec.TaskRef, tr.Namespace, serviceAccountName)
 	if err != nil {
+		logger.Error(err.Error())
```

Comment:
It should be ok if we don't log here, the returned errs will be logged eventually.